cql3/restrictions/statement_restrictions.cc

```diff
@@ -1629,5 +1655,18 @@ std::vector<query::clustering_range> statement_restrictions::get_global_index_to
     return get_single_column_clustering_bounds(options, idx_tbl_schema, *_idx_tbl_ck_prefix);
 }
 
+std::vector<query::clustering_range> statement_restrictions::get_local_index_clustering_ranges(
+        const query_options& options,
+        const schema& idx_tbl_schema) const {
+    if (!_idx_tbl_ck_prefix.has_value()) {
+        on_internal_error(
+            rlogger, "statement_restrictions::get_local_index_clustering_ranges called with unprepared index");
+    }
+
+    // Indexes are not used when multi-column restrictions are present.
+    // All restrictions should be single-column restrictions. Refs #7699.
```

Comment:
I know I asked for this comment, but I have come to realize it's not quite correct.  Let's replace it by saying that any multi-column restrictions will be handled by filtering.